internal/grpcsync/callback_serializer_test.go

```diff
@@ -28,11 +28,17 @@ import (
 	"github.com/google/go-cmp/cmp"
 )
 
+const (
+	defaultTestWatchExpiryTimeout = 100 * time.Millisecond
```

Comment:
Is this const being anywhere?

Reply:
This const was originally defined in `event_test.go` at `xdsclient` package and it is used in `callback_serializer_test.go`.
Since const values with the same meaning were not defined, I've copied here and resolved a compile error.

Reply:
The test entry point like the following is defined in `event_test.go` in the` grpcsync` package.
So `callback_serializer_test.go` depends on `event_test.go`. Should I revise this file name or do anything else?
```go
type s struct {
	grpctest.Tester
}

func Test(t *testing.T) {
	grpctest.RunSubTests(t, s{})
}
```

Also, I didn't notice that there is no usage of `defaultTestWatchExpiryTimeout`. I've deleted this definition from  `callback_serializer_test.go`.

Reply:
> Also, I didn't notice that there is no usage of defaultTestWatchExpiryTimeout. I've deleted this definition from callback_serializer_test.go.

Yes, that's all I wanted. Thanks.